openedx/adg/lms/webinars/models.py

```diff
@@ -75,7 +75,8 @@ def __str__(self):
         return self.title
 
     def clean(self):
-        if self.start_time > self.end_time:
+        super().clean()
+        if self.start_time and self.end_time and self.start_time > self.end_time:
```

Comment:
We can add future date checks for `start_time` and `end_time`  in `clean_start_date` and `clean_end_date`